layers/+email/gnus/packages.el

```diff
@@ -30,7 +30,16 @@
     :defer t
     :commands gnus
     :init
-    (spacemacs/set-leader-keys "ag" 'gnus)
+    (progn (spacemaccs/declare-prefix "ag" "+gnus" "Gnus newsreader")
```

Comment:
```suggestion
    (progn (spacemacs/declare-prefix "ag" "gnus" "Gnus newsreader")
```
Typo: double `c`
This results in an error message on startup:
>Error (use-package): gnus/:init: Symbol’s function definition is void: spacemaccs/declare-prefix

The `+` before `gnus`, results in two plus signs `++gnus` in the which-key popup.
Which-key adds the `+` automatically for prefixes.